### PR TITLE
implement rss feed button on api catlog and organzation page

### PR DESCRIPTION
**Pull request by saransh-dev**
# Fixed Issue #2508

### RSS Feed button on API catlog
rss feed button on api

### RSS Feed button on Particular organization page
rss feed button on organization

### Functionality
rss feed

Closes: #2508

### CODE REVIEW

**Comment by bajiat** [comment]
@marla-singer Could you review this PR? @brylie has already been mentoring

**Comment by marla-singer** [comment]
@saransh-dev 
Ui Tweaks
1. Misaligned elements in Organization profile. 
joxi_screenshot_1505904373688

joxi_screenshot_1505906465487

2. I think RSS button is too big
joxi_screenshot_1505904373688 1

**Comment by saransh-dev** [comment]
@marla-singer Please check the button size on api catalog page in below screenshot.

apis

File: apinf_packages/api_catalog/client/api_catalog.less [hunk 1]
```diff
@@ -33,3 +33,10 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
 .col-grid{
   margin-bottom: 1.87em;
 }
+
+.navbar-right > a.rss-feed {
+  font-size: 38px;
+  margin-top: 17px;
+  margin: 0px;
```
Comment (marla-singer) [inline apinf_packages/api_catalog/client/api_catalog.less]: controversial rules.

Leave only `padding: 0`

File: apinf_packages/organizations/client/profile/profile.css [hunk 2]
```diff
@@ -26,8 +26,13 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
 }
 .organization-header{
 	display: inline-block;
-	width: 100%;
 }
-.organization-header > a{
-	margin-top: 17px;
+.organization-header-side-icon{
+  float: right;
```
Comment (marla-singer) [inline apinf_packages/organizations/client/profile/profile.css]: Use bootstrap class `pull-right`. Don't create a new for floating

File: apinf_packages/organizations/client/profile/profile.css [hunk 3]
```diff
@@ -26,8 +26,13 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
...
+}
+.organization-header-side-icon > a {
+  display: inline-block;
```
Comment (marla-singer) [inline apinf_packages/organizations/client/profile/profile.css]: Don't create a new rule for `inline-block` displaying. Use Bootstrap classes
http://getbootstrap.com/docs/3.3/css/#forms-inline

File: apinf_packages/organizations/client/profile/profile.css [hunk 4]
```diff
@@ -26,8 +26,13 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
...
+}
+.organization-header-side-icon > ul {
+  display: inline-block;
```
Comment (marla-singer) [inline apinf_packages/organizations/client/profile/profile.css]: Don't create a new rule for inline-block displaying. Use Bootstrap classes
http://getbootstrap.com/docs/3.3/css/#forms-inline

Comment (saransh-dev) [reply apinf_packages/organizations/client/profile/profile.css]: `form-inline` is not working here. See below screen shot
org

File: apinf_packages/api_catalog/client/api_catalog.less [hunk 5]
```diff
@@ -37,6 +37,5 @@ https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence
 .navbar-right > a.rss-feed {
   font-size: 38px;
   margin-top: 17px;
```
Comment (marla-singer) [inline apinf_packages/api_catalog/client/api_catalog.less]: Remove margin-top too

File: apinf_packages/organizations/client/profile/profile.css (hunk 4 above)
Comment (marla-singer) [reply apinf_packages/organizations/client/profile/profile.css]: Maybe you wrong used `form-inline`?

**Pull request merged**